sources/src/ChatGPTPromptPopover.tsx

```diff
@@ -203,7 +204,7 @@ function Internal(
       </Box>
       {response && (
         <Box sx={{ p: 2, pt: 0, maxWidth: 500 }}>
-          <Typography sx={{ pb: 1 }} children={response} />
+          <Typography sx={{ pb: 1 }}>{response}</Typography>
```

Comment:
Please revert any of these kind of changes to minimise noise in the change history

Reply:
I rolled back this change